### DISCUSSION TITLE
Extracting Data from a point after interpolating data

### DISCUSSION THREAD

**Discussion by jcrankin293** [gempy-project/gempy#866] [Q&A]
Hello, 

Firstly, fantastic package; I have found it to be an enriching learning experience. 

I have made some models and these have interpolated borehole data to make a map. I was wondering how can I view the layers and their XYZ coordinates? 

For example, lets say I wish to see a data point halfway between two interpolated points and want to see the layer composition and their depths. 

Thank you!

**Comment by NilsChudalla** [comment]
Hi @jcrankin293

Custom Grids are perfectly for that! They are also very helpful, if you are only interested in a very specific area which is not a regular grid. Many plotting functions might work afterwards, so you will have to come up with your own way :) 
https://docs.gempy.org/tutorials/ch1_fundamentals/ch1_3a_grids.html#custom-grid

**Reply by jcrankin293** [reply]
No luck extracting so far!

When I did one interpolation with the borehole mesh, my results were rather odd; the topsoil was lower than the chalk (which I saw at +6m) but the topsoil was -36m. Is this to be expected or is there a way of grabbing a lowest value?

**Reply by javoha** [reply]
Hi, would you be able to provide your model so we can take a look?
